app/assets/stylesheets/dm/pages/_partials/_header.scss

```diff
@@ -212,6 +216,12 @@
           background-position: right 13% top 50%;
         }
 
+        button[aria-expanded="false"], button[aria-expanded="true"] {
+          &[aria-controls="xr-network-dropdown"] {
+            background-position: right 13% top 50%;
+          }
+        }
+
```

Comment:
This was not working as intended, because you have it nested inside of the
```
button[aria-expanded="false", button[aria-expanded="true"]
```
block. I found the `right: 18%` and `top: 50%` style to be a little better fit, so let's try, starting at line `214`:
```
button[aria-expanded="false"], button[aria-expanded="true"] {
  &[aria-controls="browse-by-locations-dropdown"] {
    background-position: right 13% top 50%;
  }

  &[aria-controls="dm-user-profile-options"], &[aria-controls="xr-network-dropdown"] {
    background-position: right 18% top 50%;
  }
}
```

Reply:
Got it.